app/controllers/boxes_controller.rb

```diff
@@ -57,19 +57,23 @@ def validate
 
     # TODO: should be handled by BoxForm (& duplicates BoxForm#parse_csv)
     CSV.open(params[:csv_box].path, headers: true) do |csv|
-      while csv.headers == true
-        csv.readline
-      end
```

Comment:
I understand this is related to the fix, but in my tests `CSV.open` would merely open the file and `csv.headers` would be `true` until I read at least one line, in which case it became `["Batch", ...]` :confused:

Reply:
The `csc.each do ...` Actually reads a line and solve the problem... It's failing in your local?

Reply:
No no, but that's very weird :/